.changeset/giant-suits-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-azure-devops-backend': patch
+---
+
+Fixed bug with extractPartsFromAsset that resulted in a leading "." being removed from the path in an otherwise valid path (ex. ".assets/image.png"). The leading "." will now only be moved for paths beginning with "./".
```

Comment:
```suggestion
Fixed bug with `extractPartsFromAsset` (used by the README card feature) that resulted in a leading "." being removed from the path in an otherwise valid path (ex. ".assets/image.png"). The leading "." will now only be moved for paths beginning with "./".
```

When I first saw this PR I was confused as I didn't recognize this code or why you would need it. I think adding this just helps clarify what would be impacted by this change.

Reply:
This change has been made.